bindings/python/factory.py

```diff
@@ -218,4 +218,4 @@ def classfactory(theclass, meta=None, url=None, storage=None, id=None):
         __init__=BaseExtension.__init__
     )
 
-    return type(meta.name, (theclass, BaseExtension), attr)
+    return type(meta.name, (theclass, BaseExtension), attr)
```

Comment:
End files with newline
```suggestion
    return type(meta.name, (theclass, BaseExtension), attr)
    
```